privacy/ccpa/policy_test.go

```diff
@@ -386,13 +424,13 @@ func TestBuildExt(t *testing.T) {
 			description:   "Clear - Nil",
 			noSaleBidders: nil,
 			ext:           json.RawMessage(`{"prebid":{"nosale":["a", "b"]}}`),
-			expected:      nil,
+			expected:      json.RawMessage(`{}`),
```

Comment:
Why are these cases no longer nil? Does the wrapper marshal return nil if the prebid struct is empty?

Reply:
We are only clearing out the "nosale" piece of prebid struct. Before we did not use the prebid struct, but just 'map[string]interface{}'s recursively. It is easy to test for an empty map and just return nil if the map is now empty. Now we are zeroing out the only non-nil entry in a non-nil struct, so marshal marshals the struct as '{}'. We could put some logic in to test if all the prebid struct entries are nil, and then nil out the pointer, but that seems like a lot of unnecessary work.

Reply:
Would the current logic render an empty `{}` object in the request?